packages/vsce/src/commands/inspectResourceCommandUtils.ts

```diff
@@ -233,13 +236,11 @@ async function getEntryFromInputBox(resourceType: IResourceMeta<IResource>): Pro
 
     validateInput: (value: string): string | undefined => {
       const maxLength = resourceType.maximumPrimaryKeyLength ?? constants.MAX_RESOURCE_NAME_LENGTH;
-      const tooLongErrorMessage = CICSMessages.CICSInvalidResourceNameLength.message.replace("%length%", `${maxLength}`);
+      const tooLongErrorMessage = l10n.t("Resource name must be at most {0} characters", maxLength);
```

Comment:
I see that we're stopping using several bits from CICSMessages - this is fine, let's remove them from there now to tidy up.